cmd/postgres_exporter/postgres_exporter.go

```diff
@@ -990,10 +993,14 @@ func (s *Servers) GetServer(dsn string) (*Server, error) {
 	var err error
 	var ok bool
 	errCount := 0 // start at zero because we increment before doing work
-	retries := 3
+
+	// Remove this retries and use the global connectionRetries parameter
+	// retries := 3
```

Comment:
Why retain commented code?

Reply:
It was a draft PR to show where we needed a change. 
Now removed